src/IO/WriteBufferFromAzureBlobStorage.cpp

```diff
@@ -58,8 +58,15 @@ void WriteBufferFromAzureBlobStorage::finalizeImpl()
 {
     next();
 
-    auto block_blob_client = blob_container_client->GetBlockBlobClient(blob_path);
-    block_blob_client.CommitBlockList(block_ids);
+    try
+    {
+        auto block_blob_client = blob_container_client->GetBlockBlobClient(blob_path);
+        block_blob_client.CommitBlockList(block_ids);
```

Comment:
Is it OK that we ignore exceptions from the function that contains the work "commit" in name? Does it mean that we may silently lose some data?

Reply:
I though about this, but this code can be done only in finalize. Also had an idea that it will not terminate with uncaught exception if the exception is `DB::Exception`, but was not sure and did not find the place where it is supposed to be caught, do you know?

Reply:
Ok, I found it.